### PR TITLE
folder upload

### PR DESCRIPTION
**Pull request by Bhavika222**

### CODE REVIEW

**Review by Akash-ing — APPROVED** [review]

**Review by Pranjali1999 — APPROVED** [review]

**Review by vinod2406 — APPROVED** [review]
ok

**Review by akash19972306 — APPROVED** [review]

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]
Give proper indentation and remove extra line spaces

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]
write a complete code

**Review by Priyankashelke2411 — CHANGES_REQUESTED** [review]
yes done changes but remove extra lines and give proper indentations

File: Bhavika/html/Assignment2/Assig2.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+By Some accounts,they disagreed about nearly everything during that first meeting.but by the following year they struck a <mark>partnership.</mark><br>
+Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font></p>
+They called this search engine <font color="black" style=background-color:blue>.Backrub</font>
+<hr>
+</hr>
+<h2>Initial public offering </h2><br>
+Google's Initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for30 20years,Until the year 2024.<br>
+        <center><font color="green" ><b> Thank You!!</b></font></center>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment2/Assig2.html]: check indentation

File: Bhavika/html/Assignment3/Bhavika A3_2main.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.This is home page subheading.This
+      is home page subheading.This is home page subheading.This is home page
+      subheading.This is home page subheading.
+    </p>
+  </body>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment3/Bhavika A3_2main.html]: add newline at end of file

File: Bhavika/html/Assignment3/Bhavika A3_2mi-9.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                  &nbsp;
+                 
+
+           <Ol type="i">
+                             <li>Bipin</li>
+                             <li>Priyanka</li>
+                             <li>Yashasvi</li>
+                    </Ol>
+
+
```
Comment (JafarPathan) [inline Bhavika/html/Assignment3/Bhavika A3_2mi-9.html]: check this indentation and spaces 
add newline at end of file

**Review by JafarPathan — COMMENTED** [review]
do changes in all files 
1. use 4 spaces for indentation

File: Bhavika/html/Assignment3/Bhavika A3_2mi-9.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+               &nbsp;
+                 <Ol type="i">
+                            <li>Bipin</li>
+                            <li>Priyanka</li>
+                            <li>Yashasvi</li>
+                  </Ol>
+        </ul>
+  </body>
+</html>
+
```
Comment (JafarPathan) [inline Bhavika/html/Assignment3/Bhavika A3_2mi-9.html]: do indentation 
add extra line at end of file

File: Bhavika/html/Assignment2/Assig2.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+   Working from their dorm rooms,they built a search engine that used links to determine the importance of individual pages on the <font color="blue">World Wide Web.</font></p>
+   They called this search engine <font color="black" style=background-color:blue>.Backrub</font>
+   <hr>
+   </hr>
+   <h2>Initial public offering </h2><br>
+   Google's Initial public offering (IPO) took place five years later, on 19th August 2004.<br>
+   At that time Larry Page,Sergey Brin,and Eric Schmidt agreed to work together at Google for30 20years,Until the year 2024.<br>
+         <center><font color="green" ><b> Thank You!!</b></font></center>
+ </body>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment2/Assig2.html]: do indentation

File: Bhavika/html/Assignment1/form3.html [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    <h1>Introduction<br /></h1>
+    HTML is a standard markup language for Web Pages.<br />
+    <h1>HTML Editior</h1>
+    <br />
+    <b>Notepad or TextEdit.</b><br />
+    Web pages can be created and modified by using professional HTML editior.<br />
+    However,for learning HTML we recommend a simple text editior like
+    Notepad(pc)or TextEdit(Mac).
+  </body>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment1/form3.html]: do indentation

File: Bhavika/html/Assignment1/form2.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    <h1>Introduction<br /></h1>
+    HTML is a standard markup language for Web Pages.<br />
+    <h1>HTML Editior</h1>
+    <br />
+    <b>Notepad or TextEdit.</b><br />
+    Web pages can be created and modified by using professional HTML editors.<br />
+    However,for learning HTML we recommend a simple text editior like
+    Notepad(PC)or TextEdit(Mac).
+  </body>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment1/form2.html]: do indentation

File: Bhavika/html/Assignment1/form.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+  <head>
+    <title>mypage</title>
+  </head>
+  <body>
+    <h1>HTML<br /></h1>
+    <h1>Introduction<br /></h1>
+    HTML is a standard markup language for Web Pages.
+  </body>
+</html>
```
Comment (JafarPathan) [inline Bhavika/html/Assignment1/form.html]: do indentation

**Review by JafarPathan — APPROVED** [review]

**Pull request merged**